.gitignore

```diff
@@ -25,3 +25,32 @@ _testmain.go
 
 vendor
 .cover
+
+# MacOS
+
+*.DS_Store
+.AppleDouble
+.LSOverride
+
+# Icon must end with two \r
+Icon
+
```

Comment:
OK, I understand I did not think of the implications of add one platform that will open the door for many more others. Let me update the PR.